github/Requester.py

```diff
@@ -237,7 +236,7 @@ def __requestEncode(self, cnx, verb, url, parameters, requestHeaders, input, enc
         self.__authenticate(url, requestHeaders, parameters)
         requestHeaders["User-Agent"] = self.__userAgent
         if self.__apiPreview:
-            requestHeaders["Accept"] = "application/vnd.github.moondragon+json"
+            requestHeaders["Accept"] = "application/vnd.github.polaris-preview+json"
```

Comment:
@makuk66 wondering if this is broken then too, not sure if we want to force polaris-preview for all api calls.